src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+// src/redux/books/booksSlice.js
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: { list: [] },
+  reducers: {
+    addBook: (state, action) => {
+      state.list.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.list = state.list.filter((book) => book.id !== action.payload);
```

Comment:
- If you modify the addBook action it will be necessary to modify the removeBook action, like this 👇🏽 

`state.books = state.books.filter((book) => book.id !== action.id)`

We are specifying the id of the book and the action to remove them wich have the same id. Please 🙏🏽 fix this code to be able to remove the books.